extlibs/headers/glad/include/glad/egl.h

```diff
@@ -1344,12 +1328,6 @@ static int gladLoadEGLUserPtr(EGLDisplay display, GLADuserptrloadfunc load, void
     return version;
 }
 
-static int gladLoadEGL(EGLDisplay display, GLADloadfunc load) {
-    return gladLoadEGLUserPtr(display, sf_glad_egl_get_proc_from_userptr, GLAD_GNUC_EXTENSION (void*) load);
-}
-
-
-
```

Comment:
Were these changes required, @SuperV1234 and can/should we keep them, @binary1248?

Reply:
I removed the functions in my first commit. They are not used and would cause warnings on some compilers.